app/assets/stylesheets/application.sass.scss

```
@@ -1,28 +1,112 @@
 // Entry point for your Sass build
 
-@import "palette";
-@import "bootstrapVariables";
+@import url("palette");
```

I get a 404 for this import when it's wrapped in the `url()` helper.
```suggestion
@import "palette";
```